.github/workflows/netlify.yml

```diff
@@ -0,0 +1,12 @@
+name: Re-run build from 'main' on Netlify every two hours
+on:
+  schedule:
+    - cron: '0 */2 * * *'
```

Comment:
I would run it every 6h as we have a 1000 minute/month build limit @ Netlify.

Reply:
Good thinking - thanks!